spec.bs

```diff
@@ -1387,6 +1478,227 @@ To <dfn>build an interest group passed to generateBid</dfn> given an [=interest
   1. Return |igGenerateBid|.
 </div>
 
+<div algorithm>
+To <dfn>validate and convert additional bids</dfn> given an [=auction config=] |auctionConfig|, an
+[=auction config=]-or-null |topLevelAuctionConfig|, and a [=map=] |negativeTargetInfo| whose
+[=map/keys=] are [=tuples=] consisting of an [=origin=] and a [=string=], and whose [=map/values=]
+are [=tuples=] consisting of an [=origin=] and a [=byte sequence=]:
+
+  1. Let |auctionNonce| be |auctionConfig|'s [=auction config/auction nonce=].

```

Comment:
Whenever you introduce a new algorithm, I'd recommend adding a first-line assertion about whether this is or is not running in parallel. It really helps with readability. If you could do that here and below that would be terrific.